cmd/hubproxy/main_test.go

```diff
@@ -1,153 +0,0 @@
-package main
-
-import (
-	"os"
-	"testing"
-
-	"hubproxy/internal/config"
-
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestConfigPrecedence(t *testing.T) {
```

Comment:
So this file was basically just testing the old `config.Config` parsing, but now all that is viper handled.

Reply:
:+1: